packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -24,10 +24,17 @@ enum _AnimationDirection {
 
 /// A controller for an animation.
 ///
-/// An animation controller can drive an animation forward or backward and can
-/// set the animation to a particular value. The controller also defines the
-/// bounds of the animation and can drive an animation using a physics
-/// simulation.
+/// This class lets you perform tasks such as:
+///
+/// * Play an animation [forward] or in [reverse], or [stop] an animation.
+/// * Set the animation to a specific [value].
+/// * Define the [upperBound] and [lowerBound] values of an animation.
```

Comment:
The `lowerBound` and `upperBound` properties are an alternative to creating a Tween. The properties are most commonly used with simulations with the `fling()` function.

In general, using upperBound and lowerBound shouldn't be mixed with Tweens.